docs/architecture.svg

```diff
@@ -1,2 +1,2 @@
d="M 348.92 380.29 L 343.07 385.49 L 343.85 381.66 L 341.25 378.73 Z" fill="#6c8ebf" stroke="#6c8ebf" stroke-miterlimit="10" pointer-events="none"/><path d="M 363.85 448.34 L 116.15 381.66" fill="none" stroke="#d6b656" stroke-miterlimit="10" pointer-events="none"/><path d="M 368.92 449.71 L 361.25 451.27 L 363.85 448.34 L 363.07 444.51 Z" fill="#d6b656" stroke="#d6b656" stroke-miterlimit="10" pointer-events="none"/><path d="M 111.08 380.29 L 118.75 378.73 L 116.15 381.66 L 116.93 385.49 Z" fill="#d6b656" stroke="#d6b656" stroke-miterlimit="10" pointer-events="none"/><path d="M 364.56 446.69 L 260.44 383.31" fill="none" stroke="#d6b656" stroke-miterlimit="10" pointer-events="none"/><path d="M 369.05 449.42 L 361.25 448.77 L 364.56 446.69 L 364.89 442.79 Z" fill="#d6b656" stroke="#d6b656" stroke-miterlimit="10" pointer-events="none"/><path d="M 255.95 380.58 L 263.75 381.23 L 260.44 383.31 L 260.11 387.21 Z" fill="#d6b656" stroke="#d6b656" stroke-miterlimit="10" pointer-events="none"/><path d="M 371.75 443.88 L 388.25 386.12" fill="none" stroke="#d6b656" stroke-miterlimit="10" pointer-events="none"/><path d="M 370.31 448.93 L 368.86 441.23 L 371.75 443.88 L 375.6 443.16 Z" fill="#d6b656" stroke="#d6b656" stroke-miterlimit="10" pointer-events="none"/><path d="M 389.69 381.07 L 391.14 388.77 L 388.25 386.12 L 384.4 386.84 Z" fill="#d6b656" stroke="#d6b656" stroke-miterlimit="10" pointer-events="none"/><path d="M 75 600 L 88.95 516.28" fill="none" stroke="#6c8ebf" stroke-miterlimit="10" pointer-events="none"/><path d="M 89.82 511.1 L 92.12 518.58 L 88.95 516.28 L 85.21 517.43 Z" fill="#6c8ebf" stroke="#6c8ebf" stroke-miterlimit="10" pointer-events="none"/><path d="M 220 600 L 95.24 513.62" fill="none" stroke="#6c8ebf" stroke-miterlimit="10" pointer-events="none"/><path d="M 90.92 510.64 L 98.67 511.74 L 95.24 513.62 L 94.68 517.5 Z" fill="#6c8ebf" stroke="#6c8ebf" stroke-miterlimit="10" pointer-events="none"/><path d="M 355 600 L 96.03 512.05" fill="none" stroke="#6c8ebf" stroke-miterlimit="10" pointer-events="none"/><path d="M 91.06 510.36 L 98.81 509.3 L 96.03 512.05 L 96.56 515.92 Z" fill="#6c8ebf" stroke="#6c8ebf" stroke-miterlimit="10" pointer-events="none"/><path d="M 370 503 L 393.41 593.83" fill="none" stroke="#d6b656" stroke-miterlimit="10" pointer-events="none"/><path d="M 394.72 598.92 L 389.58 593.01 L 393.41 593.83 L 396.36 591.27 Z" fill="#d6b656" stroke="#d6b656" stroke-miterlimit="10" pointer-events="none"/><path d="M 370 503 L 120.95 597.74" fill="none" stroke="#d6b656" stroke-miterlimit="10" pointer-events="none"/><path d="M 116.04 599.6 L 121.34 593.84 L 120.95 597.74 L 123.83 600.39 Z" fill="#d6b656" stroke="#d6b656" stroke-miterlimit="10" pointer-events="none"/><path d="M 370 503 L 264.78 595.79" fill="none" stroke="#d6b656" stroke-miterlimit="10" pointer-events="none"/><path d="M 260.84 599.26 L 263.77 592.01 L 264.78 595.79 L 268.4 597.26 Z" fill="#d6b656" stroke="#d6b656" stroke-miterlimit="10" pointer-events="none"/></g></svg>
```

Comment:
You should also update the source file `architecture.draw.io.xml`.

Reply:
Done! Also, during updating sources, I fixed the inconsistency for Redis between source and `svg` files.